client/my-sites/checkout/composite-checkout/wpcom/components/wp-order-review-line-items.js

```diff
@@ -53,10 +53,29 @@ function WPLineItem( {
 
 	return (
 		<div className={ joinClasses( [ className, 'checkout-line-item' ] ) }>
-			<LineItemTitle id={ itemSpanId } item={ item } />
+			<LineItemTitle id={ itemSpanId }>{ item.label }</LineItemTitle>
 			<span aria-labelledby={ itemSpanId }>
-				<LineItemPrice lineItem={ item } />
+				<LineItemPrice item={ item } />
 			</span>
+			{ item.sublabel && (
+				<LineItemMeta>
+					{ 'plan' !== item.type || ! shouldShowVariantSelector
+						? translate( '%(sublabel)s: %(interval)s', {
+								args: {
+									sublabel: item.sublabel,
+									interval: translate( 'billed annually' ),
+								},
+								comment: 'product type and billing interval, separated by a colon',
+						  } )
+						: item.sublabel }
+					{ item.wpcom_meta?.is_bundled && item.amount.value === 0 && (
+						<BundledDomainFreeUI>{ translate( 'First year free' ) }</BundledDomainFreeUI>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'First year free with your plan' )` *ES Score: 6*